README.markdown

```diff
@@ -34,25 +34,23 @@ test framework.)
 
 `blackjack.rb` contains initial work on a Blackjack simulation.
 
-Your implementation should satisfy the following use cases and Blackjack rules:
+Your implementation should complete the initial work on the simulation and satisfy the following use cases and Blackjack rules:
 
  * As a Player I can get a hand with two cards in it
  * As a Dealer I can get a hand with two cards in it
  * As a Player I can see what card the dealer is showing
- * As a Player I can bust (lose immediately) when I am getting cards
  * As a Player I can blackjack (win immediately) when I am dealt cards (this is a simplification)
+ * As a Player I can bust (lose immediately) when I am getting cards
  * As a Dealer I can draw cards after the player until I win or lose
+ * At the end of the round (after the Dealer's turn), the result of the round is determined
 
 **Rules:**
 
  * Bust - Occurs when all possible hand values are greater than 21 points
  * Blackjack - Occurs when a player or dealer is dealt an ace and a 10-point card
  * Dealer - Stays on 17 or above
+ * Round Flow - Player acts first, then the dealer acts if necessary
```

Comment:
I think Round Flow belongs with the acceptance criteria and not the **Rules**.

Reply:
I put this here for a few reasons:

1) There is already an implied round flow in the current use cases
2) I did not want to imply that a `Round` class should be implemented by adding a criterion for this
3) I do think round flow is part of the rules of blackjack. For example, vingt-et-un, which is similar to blackjack, has a different round flow.